src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import { Todo } from '../../types/Todo';
+
+type FooterProps = {
+  itemsLeft: number;
+  sortBy: string;
+  onSortBy: (sort: string) => void;
+  todos: Todo[];
+  onCleanCompleted: () => void;
+};
+
+export function Footer({
+  itemsLeft,
+  sortBy,
+  onSortBy,
+  todos,
+  onCleanCompleted,
+}: FooterProps) {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {itemsLeft} items left
+      </span>
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link ${sortBy === 'all' ? 'selected' : ''}`}
+          data-cy="FilterLinkAll"
+          onClick={e => {
+            e.preventDefault();
+            onSortBy('all');
+          }}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={`filter__link ${sortBy === 'active' ? 'selected' : ''}`}
+          data-cy="FilterLinkActive"
+          onClick={e => {
+            e.preventDefault();
+            onSortBy('active');
+          }}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={`filter__link ${sortBy === 'completed' ? 'selected' : ''}`}
+          data-cy="FilterLinkCompleted"
+          onClick={e => {
+            e.preventDefault();
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method